### ISSUE TITLE
Philosopher stone duplicator

### ISSUE THREAD

**Issue by amer780** [ChosenArchitect/Project-Architect-2#39]
**Describe the bug**
->Duplicating item 

**To Reproduce**
1. Using the philosopher stone with diamond in crafting table for getting gold
2. I shift-clicked with right-left clicked together
3. then it duplicated the phil stone
4. 

**Expected behavior**
->I don't know maybe it's Ars storage problem or projectE

**Screenshots**
If applicable, add screenshots to help explain your problem. (Use F2 to take a screenshot in-game)

**Modpack version**
->2.6
**Java Version**
->I think 21
**Modpack Launcher Used**
->The original one
**Ram Allocated**
->24GB

**Additional context**
->Nothing

https://github.com/user-attachments/assets/9a888a24-5b7b-4d7b-865d-3c99a5495c47

**Comment by 5ully** [comment]
Does the items exist or are they just desynced items (confirm it by relogging after the craft)

**Comment by amer780** [comment]
I relogged and it's already exist

**Comment by monkeybug34** [comment]
suggestion to just remove EMC value from the Philosopher's Stone. they're easy enough to craft more if you need them. doesn't fix the dupe, but does fix the potential EMC exploit.

**Comment by IRicTheCoder** [comment]
Finally figured this one out, seems to be an issue with the way the Storage Lectern pulls things from inventories and returns them. If there is a stone in the inventory and another on the crafting, doing any interactions dupes the stone.

Does not seem to be an issue with ProjectE more like an issue from Ars Noveau way of handling ingredients that remain on the grid.

**Comment by F1r3w477** [comment]
> suggestion to just remove EMC value from the Philosopher's Stone. they're easy enough to craft more if you need them. doesn't fix the dupe, but does fix the potential EMC exploit.

Really dislike this idea unless #233 can be resolved. Had to put 33000 philosophers stones into my storage for auto crafting. Definitely would not like to make them by hand or wait for auto crafting to do it.

**Comment by 5ully** [comment]
> > suggestion to just remove EMC value from the Philosopher's Stone. they're easy enough to craft more if you need them. doesn't fix the dupe, but does fix the potential EMC exploit.
> 
> Really dislike this idea unless #233 can be resolved. Had to put 33000 philosophers stones into my storage for auto crafting. Definitely would not like to make them by hand or wait for auto crafting to do it.

This is not recommended (having a bunch of philosphers stones for autocrafting in storage), but more of a limitation of RS/ae2 as the stone works similar to a full bucket. You can easily get past this by using a RFTools crafter (these are also quicker than RS/AE2 crafters).

**Comment by IRicTheCoder** [comment]
After a lot of investigation we came to the conclusion this issue is caused by the way Vanilla Minecraft implements the `Crafting Remainder` (items that stay in the grid after use) and how that is handled by mods, there are a lot more mods that have this issue and it does not happen always, depends on a few different variables.

I have seen a few reports related to this in the MC issue tracker (not specifically mod related or directly this issue but related to it), this could be something that could be fixed in the mod loader maybe, or through a specific mod. This is currently beyond our scope or ability to fully fix the issue.

I will look into potential ways of fixing it, but currently such is not possible. Closing the issue as `unable to solve`.
